src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayRent = 40;
+
+  if (days >= 7) {
+    return dayRent * days - 50;
+  }
+
+  if (days >= 3) {
+    return dayRent * days - 20;
```

Comment:
Replace the hard-coded value '20' with a constant that explains what this value represents, as per the checklist item 2. A good name for this constant could be `SHORT_TERM_DISCOUNT`.